python/lsst/ctrl/mpexec/mpGraphExecutor.py

```diff
@@ -168,11 +168,19 @@ def _executeJob(
 
         quantum = pickle.loads(quantum_pickle)
         try:
-            quantumExecutor.execute(task_node, quantum)
+            _, report = quantumExecutor.execute(task_node, quantum)
+        except Exception as exc:
+            _LOG.debug("exception from task %s dataId %s: %s", task_node.label, quantum.dataId, exc)
+            report = QuantumReport.from_exception(
+                exception=exc,
+                dataId=quantum.dataId,
+                taskLabel=task_node.label,
+            )
+            raise
```

Comment:
This is exactly the same code as was originally in `SingleQuantumExecutor`, and will fail for the same reason.

Reply:
It will fail, but failure should be handled reasonably at this level in `report()` method.

Reply:
Fixed in a commit I just force-pushed.